### PR TITLE
Add the BEST Portfolio (BEST) to Set-Uniswap Token List

### PR DESCRIPTION
**Pull request by hepnerthomas**
When submitting a new or existing set details change, make sure you have followed the following steps:

- [x]  Add a new file (or update existing file) into the `set-details` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json)
- [x] Make sure that the JSON follows the correct schema here: https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json. You can verify it by pasting the schema into https://www.jsonschemavalidator.net and see if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (can use https://hjson.github.io/try.html for example)
- [x] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [x] If adding brand new token, make sure to add it to the set.tokenlist.json as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json
- [x] Attach the proper signature (signed current version number [i.e. 1] via TokenSets UI) here (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

[ATTACH SIGNATURE HERE]

### CODE REVIEW

File: set.tokenlist.json [hunk 1]
```diff
@@ -167,11 +167,19 @@
...
+      "name": "The BEST Portfolio",
+      "symbol": "BEST",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/tokens/best.png"
+    }    
   ],
   "version": {
     "major": 0,
-    "minor": 2,
+    "minor": 3,
```
Comment (asoong) [inline set.tokenlist.json:182]: No worries i'll update these periodically.

File: set-details/0xEab53D2Dc181d5c2D316a3370471a028C621EdE7.setdetails.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "setSymbol": "BEST",
+      "setDecimals": 18,
+      "setIcon": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/tokens/best.png", 
+      "setDescription": "The ₿EST Portfolio is a set of Ethereum-based cryptoassets actively managed by Token Terrier. ₿EST seeks to accrue value over the long-term by aggressively investing in the cryptoasset ecosystem.\nRead the Frequently Asked Questions (FAQ): https://www.tokenterrier.com/best-faq ",
+      "managerName": "Token Terrier",
+      "managerIcon": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/managers/TokenTerrier.png",
+      "managerDescription": "Token Terrier LLC (“Token Terrier”) creates informational and educational content for cryptoasset investors.",
+      "managerTwitter": "TokenTerrier",
+      "website": "https://www.tokenterrier.com/best"
+
```
Comment (asoong) [inline set-details/0xEab53D2Dc181d5c2D316a3370471a028C621EdE7.setdetails.json:14]: nit: whitespace

**Review by asoong — APPROVED** [review]
Address minor comments

**Comment by hepnerthomas** [comment]
@asoong PR is ready to be merged. This was my first time doing a rebase in a pull request (used to doing merges) so apologies for the confusing series of commits in the PR.

File: set.tokenlist.json [hunk 3]
```diff
@@ -48,22 +48,246 @@
...
       "decimals": 18,
       "logoURI": "https://raw.githubusercontent.com/pro-blockchain-com/uniswap-tokenlist/main/assets/tokens/eth2xfli.svg"
+    },
+    {
+      "chainId": 1,
+      "address": "0xEab53D2Dc181d5c2D316a3370471a028C621EdE7",
+      "name": "The BEST Portfolio",
+      "symbol": "BEST",
+      "decimals": 18,
+<<<<<<< HEAD
```
Comment (asoong) [inline set.tokenlist.json:65]: Heads up

**Pull request closed without merging**